sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosConstants.java

```diff
@@ -8,11 +8,12 @@
  * This class encapsulates constants common to DC/OS and its services.
  */
 public class DcosConstants {
-    private static final String MASTER_MESOS = "master.mesos";
+    private static final String MESOS_MASTER = "master.mesos";
+    private static final String MESOS_LEADER = "leader.mesos";
 
     @SuppressWarnings("PMD.AvoidUsingHardCodedIP")
-    public static final String MESOS_MASTER_ZK_CONNECTION_STRING = MASTER_MESOS + ":2181";
-    public static final String MESOS_MASTER_URI = "http://" + MASTER_MESOS;
+    public static final String MESOS_MASTER_ZK_CONNECTION_STRING = MESOS_MASTER + ":2181";
+    public static final String MESOS_LEADER_URI = "http://" + MESOS_LEADER;
```

Comment:
I don't know.  This has worked for a very long time.  Changing to `https` sounds like a different PR to me.